### ISSUE TITLE
Overlay function is failing

### ISSUE THREAD

**Issue by dkholo** [dyerlab/popgraph#3]
> require(maps)
> map( "state" )
> overlay_popgraph(g)
Error in overlay_popgraph(g) : 
  Not functioning at this time, I'm in Europe and will fix when I return

**Comment by dyerlab** [comment]
Yes, the google maps api has changed and I’ve had to pull out some of that code.  Use geom_nodeset() and geom_edgeset() instead.
